app/communities-configs/filecoin/index.js

```diff
@@ -0,0 +1,318 @@
+import React from 'react';
+import { css } from 'styled-components';
+
+import FileiconLogo from './images/filecoin-logo.svg?inline';
+import favicon from '!file-loader?name=[name].[ext]!images/favicon-filecoin.ico';
+
+import RobotoRegularEOT from './fonts/Roboto-Regular.eot';
+import RobotoRegularWOFF from './fonts/Roboto-Regular.woff';
+import RobotoRegularTTF from './fonts/Roboto-Regular.ttf';
+
+import CustomSubHeader from 'components/CustomSubHeaders/CustomSubHeader';
+import CustomMobileSubHeader from 'components/CustomSubHeaders/CustomMobileSubHeader';
+
+const domainName = 'https://filecoin.io/';
+
+const colors = {
+  firm: '#0090FF',
+  elText: '#000000',
+  mainText: '#7A7A7A',
+  link: '#7A7AF0',
+  bg: '#FFFFFF',
+  extraBg: '#F3F3F3',
+  menuItemText: '#999999',
+  selectedMenuItemText: '#000000',
+};
+
+export const CustomSubHeaderConfig = {
+  design: 'filecoin_style',
+  styles: {
+    bg: {
+      header: colors.bg ?? '#FFFFFF',
+      dropdown: colors.bg ?? '#FFFFFF',
+    },
+    color: {
+      a: colors.menuItemText ?? 'rgba(0,0,0,0.6)',
+    },
+    font: {
+      body: 'Roboto-Regular, Arial, sans-serif',
+    },
+    header: {
+      background: colors.bg ?? '#ffffff',
+    },
+    subHeader: `
+      border-bottom: 1px solid #CECECE;
+      a, span {
+        transition: all .3s ease-out;
+      }
+      a:hover, span:hover {
+        color: ${colors.selectedMenuItemText}!important;
+        opacity: 1!important;
+      }
+    `,
+    subitems: `
+      left: 50%!important;
```

Comment:
please use `css` from styled-components

Reply:
and check all places

Reply:
> please use `css` from styled-components

I didn't quite understand what was meant. This properties will be inserted as strings in styled-components in common components. I took this pattern from community config for another community.